test/Modal/Modal_test.js

```diff
@@ -0,0 +1,54 @@
+var assert = require('assert');
```

Comment:
Do you think we'll need more than 1 test file per component? If not we probably don't need a separate directory for each one